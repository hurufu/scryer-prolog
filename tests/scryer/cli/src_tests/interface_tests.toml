args = [
    "-f",
    "--no-add-history",
    "-g", "halt",
    "src/tests/interface/test.pl"
]
