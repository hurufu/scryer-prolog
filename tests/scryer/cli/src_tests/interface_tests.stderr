% Warning: Head hello(b,A) doesn't unify with any of [hello(a,B)] at line 7 of test.pl
